src/lib.rs

```diff
@@ -152,17 +151,17 @@ pub fn run(config: &Config) -> Result<(), Box<Error>> {
     Ok(())
 }
 
-/// Performs a case sensitive search on `text`, looking for `pattern` matches. If a match is found
-/// on a line, the whole line is returned.
-pub fn search_case_sensitive<'a>(pattern: &str, text: &'a str) -> Vec<&'a str> {
-    text.lines().filter(|line| line.contains(pattern)).collect()
-}
+/// Performs a regular expression search on `text`, looking for `pattern` matches. Case sensitivity is
+/// controlled by the `case_insensitive` boolean flag. If a match is found on a line, 
```

Comment:
`case_insensitive` should be `case_sensitive`, as used on the function signature.